lib/tasks/bower-install.js

```diff
@@ -12,13 +12,20 @@ module.exports = Task.extend({
   },
   // Options: Boolean verbose
   run: function(options) {
+    var existsSync     = require('exists-sync');
     var chalk          = require('chalk');
     var bower          = this.bower;
+    var bowerJson      = path.join(this.project.root, 'bower.json');
     var bowerConfig    = this.bowerConfig;
     var ui             = this.ui;
     var packages       = options.packages || [];
     var installOptions = options.installOptions || { save: true };
 
+    if (!existsSync(bowerJson)) {
+      ui.writeWarnLine('Skipping bower install: bower.json not found');
```

Comment:
I believe this should just throw, as the command failed and we likely want the status code to reflect this.